packages/field-plugin/src/createFieldPlugin/createPluginActions/createPluginActions.test.ts

```diff
@@ -231,9 +231,29 @@ describe('createPluginActions', () => {
         field: 'dummy',
         callbackId: TEST_CALLBACK_ID,
         filename,
+        fieldtype: 'asset',
+        name: '',
+        meta_data: {},
```

Comment:
Just a question: Since metadata has already title, source, and such, why do we need the properties twice?

Reply:
I'm not 100% sure, but I think it can contain more than that.

![Screenshot 2024-09-17 at 14 12 47](https://github.com/user-attachments/assets/4f81b6be-a63b-4d69-b91f-9c45a15044fe)

https://www.storyblok.com/docs/api/management/core-resources/assets/the-asset-object